src/System.DirectoryServices/src/System/DirectoryServices/DirectoryDescriptionAttribute.cs

```diff
@@ -1,47 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@ianhays / @tquerec, this attribute file isn't being built into the product, but it is public surface area.  Is it needed?